app/tests/lib/helpers.js

```diff
@@ -117,7 +117,6 @@ define(function (require, exports, module) {
   }
 
   function toSearchString(obj) {
```

Comment:
Minor nit: Does this need to support building out arrays in query string? Or is that an edge case?